client/functions.lua

```diff
@@ -362,6 +362,16 @@ function QBCore.Functions.GetBoneDistance(entity, Type, Bone)
     return #(boneCoords - playerCoords)
 end
 
+QBCore.Functions.GetStreetLabel = function()
```

Comment:
I think having the options to have a coords parameter would make this more versatile and you can default to the player's coords if it is not specified